### DISCUSSION TITLE
Storing a link from a field and visit it later

### DISCUSSION THREAD

**Discussion by AnushaRQA** [cypress-io/cypress#18730] [Questions and Help]
Hello all, 

I'm working on a scenario where i need to take the https link generated in a field , store it in a constant and then later visit the link. 

I tried all possible ways, but something is not right. 

I used the following snippet - 

In Test 1 :
cy.get('[data-cy=fieldName]')
        .invoke('val')
        .should('not.be.empty')
        .as('Link');

      const usablelink = this.Link;

In test 2 : 
cy.visit(usablelink);

Could someone please help me how i can do this?

**Comment by AnushaRQA** [comment]
@jennifer-shehane , could you please help?

**Comment by bahmutov** [comment]
Please read the documentation for cy.as command. All aliases are reset before each test so you cannot save the link value in Ana alias. You can store the value in the simple local variable in the spec file though
